lib/src/data/datasource/mappers/movie_mapper.dart

```diff
@@ -0,0 +1,23 @@
+import '../../../domain/entity/movie_entity.dart';
+import '../../model/movie_data.dart';
+
+class MovieMapper {
+  static Movie fromDataToEntity(MovieData movieData) {
+    return Movie(
+      title: movieData.title,
+      originalTitle: movieData.originalTitle,
+      overview: movieData.overview,
+      releaseDate: movieData.releaseDate,
+      voteAverage: movieData.voteAverage,
+      genres: movieData.genres,
+      poster: movieData.poster,
+      backdrop: movieData.backdrop,
+      originalLanguage: movieData.originalLanguage,
+      popularity: movieData.popularity,
+      id: movieData.id,
+      hasVideo: movieData.hasVideo,
+      voteCount: movieData.voteCount,
+      isAdult: movieData.isAdult,
+    );
+  }
```

Comment:
Same as the comment for the genres mapper

Reply:
fixed